modern-cpp-font-lock.el

```diff
@@ -147,27 +166,30 @@ http://en.cppreference.com/w/cpp/language/integer_literal"
         (,literal-dec-regexp (1 font-lock-constant-face)
                              (2 font-lock-keyword-face))))))
 
```

Comment:
`modern-c++-font-lock-keywords` stores the keywords generating from `modern-c++-generate-font-lock-keywords` so that the same `keywords` can be removed.